pkg/generator/batch_generator_test.go

```diff
@@ -0,0 +1,67 @@
+// Copyright © 2020 The Knative Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package generator_test
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"gotest.tools/assert"
+	"knative.dev/kperf/pkg/generator"
+)
+
+func TestBatchGenerator(t *testing.T) {
+
+	generateFunc := func(ns string, index int) (string, string) {
+		return ns, fmt.Sprintf("%s-%d", ns, index)
+	}
+	postGeneratorFunc := func(ns, name string) error {
+		return nil
+	}
+
+	t.Run("no need to create ksvc", func(t *testing.T) {
```

Comment:
Please provide more detail message, just "no need to create ksvc" is not very clear.